src/base/bittorrent/session.cpp

```diff
@@ -2413,13 +2413,30 @@ void Session::networkConfigurationChange(const QNetworkConfiguration& cfg)
     }
 }
 
+bool Session::addressPatternMatches(const QString& ipAddress, const QString& pattern)
```

Comment:
The ampersand should appear directly before the variable name, rather than after the variable type.

i.e. `const QString &ipAddress`